lms/static/sass/base/_base.scss

```diff
@@ -103,7 +103,7 @@ a:link, a:visited {
   }
 
   &:disabled, &.is-disabled, &.disabled {
-    pointer-events: none;
+    @extend %ui-disabled;
```

Comment:
NOTE: while I was borrowing base styles for the student notes specific rules added in this PR, I cleaned this up to use our disabled state extend.